src/Keboola/DbExtractor/TableResultFormat/ForeignKey.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\TableResultFormat;
+
+use Keboola\DbExtractor\Exception\UserException;
+
+class ForeignKey
+{
+    private string $name;
+
+    private string $refSchema;
+
+    private string $refTable;
+
+    private string $refColumn;
+
+    private array $requiredParams = ['name', 'refSchema', 'refTable', 'refColumn'];
+
+    public function setName(string $name): self
+    {
+        $this->name = $name;
+        return $this;
+    }
+
+    public function setRefSchema(string $refSchema): self
+    {
+        $this->refSchema = $refSchema;
+        return $this;
+    }
+
+    public function setRefTable(string $refTable): self
+    {
+        $this->refTable = $refTable;
+        return $this;
+    }
+
+    public function setRefColumn(string $refColumn): self
+    {
+        $this->refColumn = $refColumn;
+        return $this;
+    }
+
+    public function getOutput(): array
+    {
+        $ret = [];
+        foreach (get_class_vars(self::class) as $property => $propertyValue) {
+            $propertyValue = $this->{$property} ?? null;
```

Comment:
je to potreba?  kdy `$this->property` nema value je to `null` automaticke ne?

Reply:
v php 7.4 když jsem zkusil `$this->property` tak jsem dostal chybu
![image](https://user-images.githubusercontent.com/12143866/70793722-3fc1f180-1d9c-11ea-8a2e-dc11ee3aa44d.png)